src/Utilities/ToolLocationHelper.cs

```diff
@@ -105,13 +105,18 @@ public enum TargetDotNetFrameworkVersion
         /// </summary>
         Version48 = 14,
 
+        /// <summary>
+        /// version 4.8.1
+        /// </summary>
+        Version481 = 15,
+
         /// <summary>
         /// The latest version available at the time of major release. This
         /// value should not be updated in minor releases as it could be a
         /// breaking change. Use 'Latest' if possible, but note the
         /// compatibility implications.
         /// </summary>
-        VersionLatest = Version48,
+        VersionLatest = Version481,
```

Comment:
The comment indicates we shouldn't change this. Do we need to? If so should the comment be updated?

Reply:
Presumably not. I'm trying to see what breaks in unit tests by forcing this